apps/src/templates/studioHomepages/CourseBlocksGradeBands.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import ContentContainer from '../ContentContainer';
+import ResourceCard from './ResourceCard';
+import styleConstants from '../../styleConstants';
+import i18n from "@cdo/locale";
+
+const pegasusContentWidth = styleConstants['content-width'];
+
+const styles = {
+  container: {
+    width: pegasusContentWidth,
+    display: "flex",
+    justifyContent: "space-between"
+  }
+};
+
+const CourseBlocksGradeBands = React.createClass({
```

Comment:
I'm doing them all at once in a big upgrade PR - there's a `create-react-class` package that can be swapped in as a stopgap while we gradually convert everything.  It's helpful to use classes or functions going forward though.